frontend/leaderboard/src/Main.re

```diff
@@ -14,4 +14,10 @@ let blocks =
     files,
   );
```

Comment:
All the metrics-calculating functions seem to have to index into `block.data.newBlock` repetitively, maybe we can have the blocks just be that in the first place.